packages/server/graphql/queries/helpers/suggestedIntegrationHelpers.ts

```diff
@@ -114,8 +119,24 @@ export const getPermsByTaskService = async (
       .default(null)
   ])
 
+  //TODO: not ideal. Need to understand rethink schema.
+  const [azureDevopsAuths] = await Promise.all([
+    dataLoader.get('azureDevopsAuthByUserId').load(userId),
+    r
```

Comment:
can remove this line. way back we tried keeping all integrations in a single `Provider` table, but as it turns out each provider is so different that we broke them out into their own tables.  Still need to rename `Provider` to `GitHub` or similar.